apis/extension/elastic_quota.go

```diff
@@ -28,33 +28,34 @@ import (
-	AnnotationResourceKeys          = QuotaKoordinatorPrefix + "/resource-keys"
-	AnnotationTotalResource         = QuotaKoordinatorPrefix + "/total-resource"
-	AnnotationUnschedulableResource = QuotaKoordinatorPrefix + "/unschedulable-resource"
-	AnnotationQuotaNamespaces       = QuotaKoordinatorPrefix + "/namespaces"
-	AnnotationGuaranteed            = QuotaKoordinatorPrefix + "/guaranteed"
-	AnnotationAllocated             = QuotaKoordinatorPrefix + "/allocated"
-	AnnotationNonPreemptibleRequest = QuotaKoordinatorPrefix + "/non-preemptible-request"
-	AnnotationNonPreemptibleUsed    = QuotaKoordinatorPrefix + "/non-preemptible-used"
-	AnnotationAdmission             = QuotaKoordinatorPrefix + "/admission"
+	SystemQuotaName                      = "koordinator-system-quota"
+	RootQuotaName                        = "koordinator-root-quota"
+	DefaultQuotaName                     = "koordinator-default-quota"
+	QuotaKoordinatorPrefix               = "quota.scheduling.koordinator.sh"
+	LabelQuotaIsParent                   = QuotaKoordinatorPrefix + "/is-parent"
+	LabelQuotaParent                     = QuotaKoordinatorPrefix + "/parent"
+	LabelAllowLentResource               = QuotaKoordinatorPrefix + "/allow-lent-resource"
+	LabelQuotaName                       = QuotaKoordinatorPrefix + "/name"
+	LabelQuotaProfile                    = QuotaKoordinatorPrefix + "/profile"
+	LabelQuotaIsRoot                     = QuotaKoordinatorPrefix + "/is-root"
+	LabelQuotaTreeID                     = QuotaKoordinatorPrefix + "/tree-id"
+	LabelQuotaIgnoreDefaultTree          = QuotaKoordinatorPrefix + "/ignore-default-tree"
+	LabelPreemptible                     = QuotaKoordinatorPrefix + "/preemptible"
+	LabelAllowForceUpdate                = QuotaKoordinatorPrefix + "/allow-force-update"
+	AnnotationSharedWeight               = QuotaKoordinatorPrefix + "/shared-weight"
+	AnnotationRuntime                    = QuotaKoordinatorPrefix + "/runtime"
+	AnnotationRequest                    = QuotaKoordinatorPrefix + "/request"
+	AnnotationChildRequest               = QuotaKoordinatorPrefix + "/child-request"
+	AnnotationResourceKeys               = QuotaKoordinatorPrefix + "/resource-keys"
+	AnnotationTotalResource              = QuotaKoordinatorPrefix + "/total-resource"
+	AnnotationUnschedulableResource      = QuotaKoordinatorPrefix + "/unschedulable-resource"
+	AnnotationQuotaNamespaces            = QuotaKoordinatorPrefix + "/namespaces"
+	AnnotationGuaranteed                 = QuotaKoordinatorPrefix + "/guaranteed"
+	AnnotationAllocated                  = QuotaKoordinatorPrefix + "/allocated"
+	AnnotationNonPreemptibleRequest      = QuotaKoordinatorPrefix + "/non-preemptible-request"
+	AnnotationNonPreemptibleUsed         = QuotaKoordinatorPrefix + "/non-preemptible-used"
+	AnnotationAdmission                  = QuotaKoordinatorPrefix + "/admission"
+	AnnotationCheckMaxGeUsedResourceKeys = QuotaKoordinatorPrefix + "/check-max-ge-used-resource-keys"
```

Comment:
To revise the naming, how about `restricted-max-resource-keys`? WDYT

Reply:
I think `restricted-max` does not exactly express the constraint when setting the annotation.
Since `ge` for "greater than or equal to" is commonly used, `check-max-ge-used-resource-keys` may be a good but not excellent choise.

Reply:
This naming is kind of verbose...

Reply:
How about `max-strict-check-resource-keys`

Reply:
OK

Reply:
Done @saintube